test/admin.test.js

```diff
@@ -11,7 +11,10 @@ const request = require('supertest'),
   correctPassword = 'password',
   correctEmail = 'email@wolox.com.ar',
   correctAdminEmail = 'emailAdmin@wolox.com.ar',
-  badToken = 'token';
+  badToken = 'token',
+  { notifySignUp } = require('../app/helpers/mailer');
+jest.mock('../app/helpers/mailer');
+notifySignUp.mockResolvedValue(true);
```

Comment:
Other test files should not depend on this being mocked here. If you ever need to change it, you might have trouble finding it. `app.spec.js` is a better fit for this.